author.php

```diff
@@ -1,22 +1,24 @@
 <?php
-	get_header();
-	$author_image = get_avatar( get_queried_object()->ID, 'squared' );
+get_header();
+$author_image_url = get_avatar_url(get_queried_object()->ID, 'squared');
```

Comment:
Let's create some context variables for re-use throughout the template code.

```suggestion
$author_object = get_queried_object();
$author_meta = get_user_meta(get_queried_object()->ID);

$author_id = $author_object->ID;
$author_nicename = $author_object->user_nicename;
$author_first_name = $author_meta["first_name"];
$author_position = $author_meta["cc_position"];
$author_description = $author_meta["description"];
$author_image_url = get_avatar_url($author_id, 'squared');
```